### DISCUSSION TITLE
PICO with ILI9486 8bit-parallel

### DISCUSSION THREAD

**Discussion by Paul-117** [Bodmer/TFT_eSPI#1740] [Q&A - General]
Hi, 
I quite a beginner, but really want to get the ILI9486 8bit-parallel Display to work on the PICO. 
the Display I'm using is: 
http://lcdwiki.com/3.5inch_Arduino_Display-UNO
I hooked everything up according to the Setup103_RP2040_ILI9486_parallel.h file, 

Installed the Pico to the Arduino IDE following: 
https://github.com/earlephilhower/arduino-pico

The Read_User_Setup outputs following:
![Screenshot 2022-03-30 150221](https://user-images.githubusercontent.com/76645253/160843563-defcd980-4a1b-42c9-a955-b93886936f93.png)

However running any example leaves the Display white

I changed the local select user setup file to include the 103_RP2040_ILI9486_parallel.h 
Is there something I'm overlooking ?

**Comment by Bodmer** [comment]
The setup looks good.

The may be a problem with the way the board is connected to the Pico. List all power and signal connections made to the display, including those between the pico and display. 
Provide a picture of you display and pico showing connections. You can drop an image in the text box here.

**Reply by Paul-117** [reply]
Thanks for the quick reply

the setup is following:

![image](https://user-images.githubusercontent.com/76645253/161013209-e7061e94-ed24-4c77-9cff-a4c1359dc393.png)

I also tried using the 5v pin with the 3.3v with same result 

with Jumpers it is a bit messy: 

![IMG_20220331_102747](https://user-images.githubusercontent.com/76645253/161013710-6544625a-ecfd-46f7-8d56-2b4c465098ff.jpg)

![IMG_20220331_102804](https://user-images.githubusercontent.com/76645253/161013736-c4a55b5d-dbf5-4ee6-bcc3-3cf1091e9517.jpg)

**Reply by Bodmer** [reply]
Typically, the display does not use the 3.3V pin for power. The display has a 3V3 regulator fitted so you need to power with 5V. The VBUS (corner pin on Pico) can be used if the Pico is powered by USB.

**Reply by Paul-117** [reply]
sometimes the solution is that easy, it all works! 

sorry to bother you with that kind of things. This is my second TFT Display. 
Thanks for the help and for sharing the awsome Library! 
have a nice day, mine sure got a lot better
